NumLib/Fem/CoordinatesMapping/NaturalCoordinatesMapping-impl.h

```diff
@@ -28,6 +32,7 @@ template <class T_MESH_ELEMENT, class T_SHAPE_FUNC, class T_SHAPE_MATRICES>
 inline void computeMappingMatrices(
         const T_MESH_ELEMENT &/*ele*/,
         const double* natural_pt,
+        const MeshLib::ElementCoordinatesMappingLocal &/*ele_local_coord*/,
```

Comment:
The `ElementCoordinatesMappingLocal` is constructed using an element, isn't it? And an element is given as the first argument of `computeMappingMatrices()`.

Maybe a stupid question: Does it make sense to construct the object of class `ElementCoordinatesMappingLocal` within `computeMappingMatrices()`?